.editorconfig

```diff
@@ -0,0 +1,7 @@
+root=true
+
+[*]
+charset=utf8
+indent_style=space
+indent_size=4
+
```

Comment:
Any particular reason why? It will be helpful to other committers and it's a [standard](http://editorconfig.org/)